pqdb-frontend/src/App.js

```diff
@@ -81,49 +93,77 @@ class App extends React.Component {
     xhr.send();
   }
 
+  switchView(view) {
+    const history = this.props.history;
+    history.replace({
```

Comment:
I'd prefer adding history entries I think. Did you use replace for a reason?

Reply:
No, we can use push here.